sdcm/auto_ssh.py

```diff
@@ -52,4 +52,4 @@ def stop_auto_ssh(docker_name, node):
 
     LOGGER.debug("killing {docker_name}-{host_name}-autossh".format(**locals()))
```

Comment:
The debug message is still killing.

Reply:
that's still true, event that we are doing more then killing :)